app/assets/javascripts/admin/addon/components/chart.gjs

```diff
@@ -5,12 +5,9 @@ import loadScript from "discourse/lib/load-script";
 // args:
 // chartConfig - object
 export default class Chart extends Component {
-  renderChart = modifier((element) => {
+  renderChart = modifier((element, [chartConfig]) => {
```

Comment:
We don't need to pass in the argument, those are autotracked. It didn't work here because it was accessed in an async function that wasn't awaited on (and we can't await because we need the modifier to return a callback immediately)

The fix is to access the arg before the async code:
 
```suggestion
  renderChart = modifier((element) => {
    const { chartConfig } = this.args;
```